jsonnet/kube-prometheus/main.libsonnet

```diff
@@ -19,6 +19,8 @@ local utils = import './lib/utils.libsonnet';
     common: {
       namespace: 'default',
       platform: null,
+      scrapeConfigReloader: true,
+      reloaderPort: 8080,
```

Comment:
This value is specific to each component so I wouldn't define its value globally but rather per component.